aclk/legacy/agent_cloud_link.c

```diff
@@ -1450,3 +1450,53 @@ int legacy_aclk_update_alarm(RRDHOST *host, ALARM_ENTRY *ae)
 
     return 0;
 }
+
+char *legacy_aclk_state(void)
+{
+    BUFFER *wb = buffer_create(1024);
+    char *ret;
+
+    buffer_strcat(wb,
+        "ACLK Available: Yes\n"
+        "ACLK Implementation: Legacy\n"
+        "Claimed: "
+    );
+
+    char *agent_id = is_agent_claimed();
+    if (agent_id == NULL)
+        buffer_strcat(wb, "No\n");
+    else {
+        buffer_sprintf(wb, "Yes\nClaimed Id: %s\n", agent_id);
```

Comment:
In it's json counterpart, `claimed-id` will be printed (`"claimed-id":null`) even if not claimed, while here it will be only visible if `agent_id` is not null. Don't know if we should also include it here too even when no claim_id exists? (Or remove it also from json when it's not claimed) ? Just to have similar output for both. i.e.:

```
# ../install_netdata_11462/netdata/usr/sbin/netdatacli aclk-state
ACLK Available: Yes
ACLK Implementation: Next Generation
Claimed: No
Online: No
# ../install_netdata_11462/netdata/usr/sbin/netdatacli aclk-state-json
{"aclk-available":true,"aclk-implementation":"Next Generation","agent-claimed":false,"claimed-id":null,"online":false}
```

Reply:
I wanted JSON part to have fixed list of keys on purpose to aid using JSON versions in user scripts etc.
Also the JSON version will be used as new `/api/v1/aclk-state` HTTP API in future PRs (after more info is added, e.g. chart streaming state, alerts streaming state etc.)